common/lib/xmodule/xmodule/course_module.py

```diff
@@ -919,7 +919,7 @@ class CourseModule(CourseFields, SequenceModule):  # pylint: disable=abstract-me
     """
 
 
-class CourseDescriptor(CourseFields, LicenseMixin, SequenceDescriptor):
+class CourseDescriptor(CourseFields, SequenceDescriptor, LicenseMixin):
```

Comment:
Yes.

There was confusion about what being an `XBlockMixin` means, and where you have to put it in relation to an `XBlock` in the inheritance hierarchy.